docs/source/tutorials/hemeraIn5min.rst

```diff
@@ -46,11 +46,13 @@ Use :command:`git` to obtain the source and use the current ``dev`` branch and p
 
   mkdir -p ~/src
   git clone https://github.com/ComputationalRadiationPhysics/picongpu ~/src/picongpu
+  git checkout dev
   cd ~/src/picongpu
   git checkout dev
 
 .. note::
    If you get the error ``git: command not found`` load git by invoking ``module load git`` and try again.
+   The change to the ``dev`` branch is currently needed due to a CUDA driver update on hemera. 
```

Comment:
IMO we need to be more precise.

```
Attention, the example uses the dev branch instead of the latest stable release ``master``. Due to driver changes on hemera the master branch modules configuration is outdated.
```